### DISCUSSION TITLE
500 Internal server error

### DISCUSSION THREAD

**Discussion by Toyurc** [vercel/next.js#13640] [Help]
Hi, when I visit mysite/_next/static/,  I get a 500 error instead of a 404 page not found, I am currently using nextjs9.0.0.

Any ideas on how to fix this

**Comment by rafaelalmeidatk** [comment]
Hello! Can you share a repo with a minimal reproduction?

**Reply by Toyurc** [reply]
So I found a prior issue that claims to fix it, https://github.com/vercel/next.js/issues/8563. I hope it gives an idea to my problem
